qiskit/circuit/library/bit_flip_oracle.py

```diff
@@ -64,11 +64,15 @@ def __init__(
             label: A label for the gate to display in visualizations. Per default, the label is
                 set to display the textual represntation of the boolean expression (truncated if needed)
         """
-        self.boolean_expression = BooleanExpression(expression, var_order=var_order)
-
```

Comment:
You should also update the typehint `expression: str | BooleanExpression` above (I can't comment on the relevant line), like you do in other placds

Reply:
Also, very minor: should we now modify the description ``expression: A Python-like boolean expression.`` (here and in the two other places).

Reply:
Thanks, both done.